shared/fs/filepreview/text-view.native.js

```diff
@@ -15,18 +15,20 @@ const TextView = ({url, onInvalidToken, onLoadingStateChange}: TextViewProps) =>
 // provides the top and bottom margin that blends with the rest of the app.
 const webviewCSS = `
 html{
-  background-color: ${globalColors.blue5};
+  background-color: ${Styles.globalColors.blue5};
   padding-top: 32; 
   padding-bottom: 32; 
   margin: 0;
 }
 body{
-  background-color: ${globalColors.white};
-  padding: ${globalMargins.medium};
+  background-color: ${Styles.globalColors.white};
+  padding: ${Styles.globalMargins.medium};
   margin: 0;
-  color: ${globalColors.black_50};
+  color: ${Styles.globalColors.black};
   font-size: 15;
   line-height: 1.6;
+  font-family: ${Styles.globalStyles.fontTerminal.fontFamily}
```

Comment:
Do we need to worry about whether this font is available from webview on systems where this font is not installed?